+++
# Display name
name = "Skene Lab"

# Username (this should match the folder name)
authors = ["admin"]

# Is this the primary user of the site?
superuser = true

# Role/position
role = "Neurogenomics Lab"

# Organizations/Affiliations
#   Separate multiple entries with a comma, using the form: `[ {name="Org1", url=""}, {name="Org2", url=""} ]`.
organizations = [ { name = "UK Dementia Research Institute", url = "https://ukdri.ac.uk/" },
  { name = "Imperial College", url = "https://www.imperial.ac.uk/dementia-research-institute/" }]

# Short bio (displayed in user profile at end of posts)
bio = "My research interests include human genetics, single cell -omics and neurobiology."

# Enter email to display Gravatar (if Gravatar enabled in Config)
email = ""

# List (academic) interests or hobbies
#interests = [
#    "Neurobiology",
#    "Human Genetics",
#    "Psychiatric Disorders",
#    "Transcriptomics",
#    "Bioinformatics"
#  ]

  

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
#user_groups = ["Researchers", "Visitors"]
#user_groups = ["Principal Investigators"]
#user_groups = ["Principal Investigators"]
#user_groups = ["Principal Investigators"]

# List qualifications (such as academic degrees)
#[[education.courses]]
#  course = "PhD in Molecular Biology"
#  institution = "Wellcome Trust Sanger Institute, University of Cambridge"
#  year = 2014
#
#[[education.courses]]
#  course = "MPhil in Computational Biology"
#  institution = "University of Cambridge"
#  year = 2009
#
#[[education.courses]]
#  course = "BSc in Artificial Intelligence and Cybernetics"
#  institution = "University of Reading"
#  year = 2008

# Social/Academic Networking
# For available icons, see: https://sourcethemes.com/academic/docs/widgets/#icons
#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
#   form "mailto:your-email@example.com" or "#contact" for contact widget.

[[social]]
  icon = "envelope"
  icon_pack = "fas"
  link = "n.skene@imperial.ac.uk"  # For a direct email link, use "mailto:test@example.org".

[[social]]
  icon = "twitter"
  icon_pack = "fab"
  link = "https://twitter.com/n_skene"

[[social]]
  icon = "google-scholar"
  icon_pack = "ai"
  link = "https://scholar.google.co.uk/citations?user=nu6vg78AAAAJ&hl=en&oi=ao"

[[social]]
  icon = "github"
  icon_pack = "fab"
  link = "https://github.com/NathanSkene"

# Link to a PDF of your resume/CV from the About widget.
# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
# [[social]]
#   icon = "cv"
#   icon_pack = "ai"
#   link = "files/cv.pdf"

+++

Our research focuses on simple questions about complex brain traits: where do they occur (in which cell types)? When do they occur (at which developmental stages)? How do they occur (which quantitative traits of cells are affected)? We aim to develop the methods and datasets required to enable these questions to be answered systematically for any trait for which well-powered genetic data is available.

The most complete and unbiased data source currently available for the brain comes from single cell RNA-sequencing: an atlas of almost every cell type found in the central, peripheral and enteric nervous system is <a href="http://mousebrain.org/">available</a>. Using this dataset and other's like it we have investigated the cell types underlying <a href="https://www.nature.com/articles/s41588-018-0129-5.pdf">Schizophrenia</a>, <a href="https://www.biorxiv.org/content/10.1101/528463v1">Alzheimer's disease, Parkinsons, Stroke</a> and <a href="https://www.nature.com/articles/s41588-018-0152-6">cognitive ability</a>. To get greater accuracy in interpreting genetic data we need to go beyond mouse single cell RNA-seq data though: we plan to obtain human cell atlases using ATAC-seq and other genomic techniques.

Using transcriptomic datasets mapping expression changes across the lifespan we have investigated why the <a href="https://elifesciences.org/articles/17915">onset of psychiatric diseases</a> occurs in early adulthood (rather than in infancy, as would be expected for heavily genetic disorders). Currently this kind of data is only available for a small number of brain regions; we are looking to generate new datasets spanning the human lifespan for a range of disease relevant cell types. 

Identifying the causal cell types and developmental stages will not in itself lead to cures though. To get closer to that we need to understand what changes occur on the affected cells. Our approach is based on the notion that the causal changes associated with disease will be quantitative traits, much like height or BMI, but the important changes will be on the cellular level for specific subtypes of cells (i.e. increased spine density in Reln/Ndnf+ cortical interneurons).  To identify these changes and link them to genetics we need higher throughput biological methods. Towards this end we are building on the work of collaborators to develop <a href="https://www.biorxiv.org/content/early/2018/10/01/431957">spatial transcriptomic techniques</a> to enable simultaneous automated identification of all cell types found in tissue sections.

Looking for a postdoc, internship, masters or PhD? <a href="mailto:n.skene@imperial.ac.uk">Would be great to hear from you!</a>

Applications would be welcomed from both experimentalists and analysts. If you're background is not in biology but you're great at programming or mathematics then get in touch. Descriptions of potential projects will be put online soon: in the meanwhile, if this is what you want to work on then just get in touch.
